### PR TITLE
Add vulnerability-alerts GET endpoint

### LINKED ISSUE #1341: Support getting whether vulnerability scanning is enabled for a repository
Vulnerability alerts have been announced some time ago (https://developer.github.com/changes/2019-04-24-vulnerability-alerts/), and enabling/disabling is already implemented here.

However, some months later (see original announcements with addendum) a GET endpoint was added to get the current status of this setting for a repository.

I'll open a PR to add this shortly.

### PR DESCRIPTION
**Pull request by SanderKnape**
This PR adds the vulnerability-alerts GET endpoint.

Fixes #1341

### CODE REVIEW

**Comment by SanderKnape** [comment]
@googlebot I signed it!

File: github/repos_test.go [hunk 1]
```diff
@@ -630,6 +630,28 @@ func TestRepositoriesService_Edit_invalidOwner(t *testing.T) {
...
+		w.WriteHeader(http.StatusNoContent)
+	})
+
+	vulnerabilityAlertsEnabled, _, err := client.Repositories.GetVulnerabilityAlerts(context.Background(), "o", "r")
+	if err != nil {
+		t.Errorf("Repositories.GetVulnerabilityAlerts returned error: %v", err)
+	}
+
+	want := true
+	if !reflect.DeepEqual(vulnerabilityAlertsEnabled, want) {
```
Comment (gmlewis) [inline github/repos_test.go]: nit: this could simply be:

```go
if want := true; got != want {
```

instead of using reflect.

Comment (SanderKnape) [reply github/repos_test.go]: Thanks for the feedback @gmlewis ! I changed the line.

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**